server/tests/test_guardrail.py

```
@@ -78,6 +79,66 @@ def test_verdict_fields_populated(self):
+# ---------------------------------------------------------------------------
+# Sigma loader tests (deterministic, always run)
+# ---------------------------------------------------------------------------
+
+from utils.security.sigma_loader import load_sigma_rules, _field_to_regex, _translate_rule
+
+
+class TestSigmaLoader:
+    def test_loads_rules(self):
+        rules = load_sigma_rules()
+        assert len(rules) >= 20, f"Expected >=20 Sigma rules, got {len(rules)}"
+        for pattern, technique, rule_id, desc in rules:
```

Ruff (0.15.11)

[warning] Loop control variable `pattern` not used within loop body

Rename unused `pattern` to `_pattern`

(B007)